src/main/java/net/bestemor/villagermarket/menu/Shopfront.java

```diff
@@ -236,7 +236,8 @@ public void onClick(InventoryClickEvent event) {
                 return;
             }
 
-            if (event.getRawSlot() == event.getView().getTopInventory().getSize() - 1) {
+            if (event.getRawSlot() == event.getView().getTopInventory().getSize() - 1
```

Comment:
This will disable the back button for the edit shopfront inventory. Should be moved down to about line `260` as part of an `else if`.

Reply:
Just commited something about this, can you check? Let me know if it is okay now.